class_student.rb

```diff
@@ -1,22 +1,8 @@
-require_relative 'class_person'
+class Student
+  attr_accessor :name, :classroom
 
-class Student < Person
-  attr_reader :classroom
-
-  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
-    super(age, name: name, parent_permission: parent_permission)
-    @classroom = classroom
-  end
-
-  def play_hooky
-    '¯\\(ツ)/¯'
+  def initialize(name)
+    @name = name
+    @classroom = nil
   end
 end
-
-student = Student.new(16, 'Physics', 'Danie')
-
-puts student.name
```

Comment:
- [ ] `REAME.md` file: Also make your documentation looks more professional and descriptive, I think it would be better if you update it as you have unprossional content and wrong information for some sections, such as:
  - First, remove one `Getting Started` section as it is repeated twice.
  - After update some of its sub-section with correct and appropriate information such as how to `access` the cloned directory, `install` bundles, `run/use` your app, `run tests`,... To help you, I recommend you refer **[this gist_file](https://gist.github.com/tresorsawasawa/0eba0570695aef068a7837c349f94737)**